test/NuGet.Core.Tests/NuGet.Versioning.Test/SemanticVersionTests.cs

```diff
@@ -97,5 +97,12 @@ public void TryParseStrictReturnsFalseIfVersionIsNotStrictSemVer(string version)
             Assert.False(result);
             Assert.Null(semanticVersion);
         }
+
+        [Fact]
+        public void SemanticVersion_ParseVersion_ThrowsIfNullOrEmpty()
+        {
+            ExceptionAssert.ThrowsArgNullOrEmpty(() => SemanticVersion.Parse(null), "value");
```

Comment:
Warning: this is testing `NuGet.Versioning`, not the legacy `SemanticVersion` that is modified above.